pkg/test/exporter_collector_test.go

```diff
@@ -0,0 +1,166 @@
+// Copyright 2020 VMware, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package test
+
+import (
+	"net"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	"k8s.io/klog/v2"
+
+	"github.com/vmware/go-ipfix/pkg/collector"
+	"github.com/vmware/go-ipfix/pkg/entities"
+	"github.com/vmware/go-ipfix/pkg/exporter"
+	"github.com/vmware/go-ipfix/pkg/registry"
+)
+
+type Addr struct {
```

Comment:
why not directly use [net.Addr](https://golang.org/pkg/net/#Addr)?

Reply:
Because net.Addr is an interface, I just implement that here.

Reply:
ResolveUDPAddr and ResolveTCPAddr can help here. Those functions take a string as input and returns net.Addr interface.

Reply:
Got it. Updated. Thanks!